source/constructs/lib/admin/database/update.py

```diff
@@ -0,0 +1,81 @@
+import logging
+import boto3
+import json
+import os
+import pymysql
+
+logger = logging.getLogger('api')
+logger.setLevel(logging.INFO)
+
+
+def lambda_handler(event, context):
+    path = "1.0.0-1.0.1"
+    if "path" in event:
+        path = event["path"]
+    logger.info(f"exec path:{path}")
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

You have a log statement that might use unsanitized input. Depending on the context, this could result in:

1. A log injection attack that breaks log integrity, forges log entries, or bypasses monitors that use the logs. To increase the security of your code, sanitize your inputs before logging them. [Learn more](https://cwe.mitre.org/data/definitions/117.html)

2. A sensitive information leak that exposes users&#39; credentials, private information, or identifying information to an attacker. To preserve privacy in your code, redact sensitive user information before logging it. [Learn more](https://cwe.mitre.org/data/definitions/532.html)

Reply:
Cannot reproduce this log problem